mitx_residential_etl.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+"""
+Script to run SQL queries on mitx residential and upload them to an S3 bucket.
+"""
+
+import csv
+import json
+import os
+import subprocess
+import sys
+from datetime import datetime
+
+try:
+    import requests
+    from logbook import Logger, RotatingFileHandler
+    from sqlalchemy import create_engine
+    from sqlalchemy.sql import text
+except ImportError as err:
+    print("Failed to import module: ", err)
+    sys.exit("Make sure to install logbook, requests and sqlalchemy")
+
+datetime = datetime.now()
+date_suffix = datetime.strftime('%m%d%Y')
```

Comment:
I generally find this format hard to parse. In general my preferred date format is `%Y-%m-%d`.

Reply:
Changed